assets/scripts/streets/width.js

```diff
@@ -227,8 +60,8 @@ export function recalculateOccupiedWidth () {
     street.remainingWidth = 0
   }
 
-  buildStreetWidthMenu()
-  updateStreetMetadata(street)
+  window.dispatchEvent(new CustomEvent('stmx:width_updated'))
+  // updateStreetMetadata(street)
```

Comment:
I'm pretty sure we don't need it. I'm not seeing any issues either. I'll remove them.